ArticleTemplates/assets/scss/modules/external/_element.scss

```diff
@@ -59,6 +59,17 @@
         left: 0;
     }
 }
+@include mq($to: col2) {
+    .element-embed,
+    .element-instagram {
+        overflow-x: hidden;
+    }
+
+    .element-instagram .instagram-media,
+    .element-embed iframe {
+        width: calc(100vw - 24px) !important;
```

Comment:
Should this be base-px(2) instead, so we could potentially switch it to 8pt grid easier later?